helpers.lua

```diff
@@ -27,10 +27,8 @@ end
 local non_dialogue = {
     '^%b()$',
     '^（.-）$',
-    '^♬～$',
-    '^♬〜$',
-    '^♪～$',
-    '^～♪$',
+    '^-?%[.-%]$',
```

Comment:
This matches anything that starts and ends with `[]` square brackets + an optional dash at the front. So it matches stuff like
```
[THUNDER RUMBLING]
-[GASPS]
```

but not e.g. `[CHUCKLES] YOU WON'T MAKE IT TO SUNRISE.`